packages/client/src/components/NewArrivalsCarousel/useNewArrivals.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { apiURL } from '../../apiURL';
+
+export const useNewArrivals = () => {
+  const [movies, setMovies] = React.useState([]);
+  const [error, setError] = React.useState(null);
+
+  React.useEffect(() => {
```

Comment:
```suggestion
import { useEffect, useState } from 'react';

  useEffect(() => {
```

Reply:
Also, I recommend removing `useEffect` and running and passing `useNewArrivals` to `useEffect` function when we use it

Reply:
When I do this lint gives error that I didn't use the React so
I did in this way.